changelogs/current.yaml

```diff
@@ -152,6 +152,15 @@ new_features:
   change: |
     Added new access log command operators ``%START_TIME_LOCAL%`` and ``%EMIT_TIME_LOCAL%``,
     similar to  ``%START_TIME%`` and ``%EMIT_TIME%``, but use local time zone.
+- area: access_log
+  change: |
+    Added new JSON access log formatter implementation which provides 16-25x performance improvement over the old
+    implementation. The legacy implementation will be enabled by default for now but will be deprecated and removed
+    in a future release.
+    The :ref:`sort_properties <envoy_v3_api_field_config.core.v3.JsonFormatOptions.sort_properties>` field will
+    be ignored in the new implementation because the new implementation always sorts properties.
+    The new JSON formatter could be enabled by setting the runtime guard
```

Comment:
Alyssa said it's OK to default this to true; sorry about the churn. If it's OK with her it's OK with me.

Reply:
Oh, it would be great. I didn't see her comment, where is it?

Reply:
I found it in my email thread for this github issue. It can be hard to find things in the GH comments.  @alyssawilk wrote "+1 we default true for Envoy unless it's particular high risk (like the codec swaps) and merits internal testing first".

I'd actually say this is not as risky as a codec but it's pretty risky as borking the json output can break logging pipelines. It would be less risky if you were just using json_streamer as is since that's got some mileage now.

So my vote after re-reading her comment is to leave it false by default for a release, and then switch it.

Reply:
I am fine to wait until next release (1.32). Let leave it as false for now.